ouroboros-consensus/docs/VersioningSchemeDecision.md

```diff
@@ -0,0 +1,34 @@
+# Choosing a Versioning Schemes
+
+This document records our discussions around choosing how to version our packages.
+
+## Desiderata
+
+- *Simplicity and Familiarity*. We'd like the versioning scheme to be simple to explain, and ideally already well-established.
+
+- *Ease of execution*. We'd like the process of cutting a release to merely involve following a very simple checklist. We'd like it to require as few inputs, discussions, decisions, etc.
```

Comment:
The main thing we've discussed that is missing from the document is deciding that some packages will advance in lock-step. We had considered this partitioning:

```
ouroboros-consensus.cabal
ouroboros-consensus-diffusion.cabal
ouroboros-consensus-test.cabal
ouroboros-consensus-mock.cabal
ouroboros-consensus-mock-test.cabal
ouroboros-consensus-protocol.cabal

ouroboros-consensus-byron.cabal
ouroboros-consensus-byron-test.cabal
ouroboros-consensus-byronspec.cabal
ouroboros-consensus-cardano.cabal
ouroboros-consensus-cardano-test.cabal
ouroboros-consensus-cardano-tools.cabal
ouroboros-consensus-shelley.cabal
ouroboros-consensus-shelley-test.cabal```